nasa-gibs/worldview
# WV-2631: Fix Incorrect Info Dates

## Description
Fixes # wv-2631

The Black Marble (Annual, 2012 & 2016) and Black Marble - Nighttime Lights only (Annual, 2012 & 2016) layers have incorrect data in the info box.

It says:
Temporal Coverage: 2011-2014
Date Ranges:
2015 - 2014
2011 - 2010

It should be :
Temporal Coverage: 2012-2016
Date Ranges:
2016 - 2016
2012 - 2012

Other layers are similarly corrected, including:
- LIS_Very_High_Resolution_Lightning_Daily_Climatology_LIS_Mean_Flash_Rate
- LIS_Very_High_Resolution_Lightning_Seasonal_Climatology_LIS_Mean_Flash_Rate
- LIS_Very_High_Resolution_Lightning_Monthly_Climatology_LIS_Mean_Flash_Rate
- LIS_Very_High_Resolution_Lightning_Seasonal_Climatology_LIS_Mean_Flash_Rate
- Landsat_WELD_CorrectedReflectance_Bands157_Global_Annual

## How To Reproduce
- Load production WorldView
- Add "black marble annual"; both Normal & Nightime have the same symptoms so select either/both
- In the layer panel, click the "i" icon to View Description
- Note the Temporal Coverage
- Click *View Dates to expand the view
- Note the Date Ranges are incorrect
- Review other layers in the same manner as necessary

## How To Test

- git fetch --all
- git checkout wv-2631-incorrect-info-dates
- npm run build
- Load local WorldView
- Follow the steps above & confirm the dates are now correct.

 
@nasa-gibs/worldview
